Remove mentions of Simple Content Access (#2745)
theforeman/foreman-documentation: guides/common/modules/proc_creating-an-activation-key.adoc

[id="Creating_an_Activation_Key_{context}"]
= Creating an activation key

Subscription Manager attaches subscriptions differently depending on the following factors:

* Are there any subscriptions associated with the activation key?
* For {RHEL} 8 hosts: Is there system purpose set on the activation key?

Note that {Project} automatically attaches subscriptions only for the products installed on a host.
For subscriptions that do not list products installed on {RHEL} by default, such as the Extended Update Support (EUS) subscription, use an activation key specifying the required subscriptions.

Based on the previous factors, there is one possible scenario for subscribing with activation keys:

> * Activation key that attaches subscriptions automatically.

Reviewer: From the beginning of this file to here, (including `* Activation key that attaches subscriptions automatically.`, should be removed.

The blurb for the procedure can be something like "Create an activation key to assign various attributes to hosts at registration time."

Author: @jeremylenz I don't see the text that says `Based on the previous factors, there is one possible scenario for subscribing with activation keys:` in my text editor for this .adoc file. Also `Activation key that attaches subscriptions automatically.` says `Activation key attaches subscriptions automatically.` in my text editor. Here is the text I have before `Activation key attaches subscriptions automatically`:
![image](https://github.com/theforeman/foreman-documentation/assets/91690569/4b285e73-f2f1-4812-8a3e-a6507c47a4c0)

Author: Should all of that stay or be removed? @jeremylenz

Reviewer: All of this should be removed.

Not sure why your text editor would be showing different content than what's here. Maybe worth checking with a colleague to see if there are some git shenanigans you can do to fix it. :)

You can configure the activation key to set system purpose attributes on hosts during registration.

> This can help determine which repositories are available on the host, and also helps with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.

Reviewer: This should be moved and combined with the "Optional:" below

[id="Creating_an_Activation_Key_{context}"]

> = Creating an Activation Key

Reviewer 2: We are opting for sentence case instead of title case for titles and headings. https://community.theforeman.org/t/sentence-case-for-foreman-guide-titles-and-headings/36565/1

We can start applying here as well.

You can use activation keys to define a specific set of subscriptions to attach to hosts during registration.

> The subscriptions that you add to an activation key must be available within the associated Content View.

Reviewer: This is no longer true. This should be deleted.

You can use activation keys to define a specific set of subscriptions to attach to hosts during registration.

> The subscriptions that you add to an activation key must be available within the associated Content View.

Reviewer: I see the previous comment was marked resolved, but this is still not deleted. Need to remove these sentences:

```
You can use activation keys to define a specific set of subscriptions to attach to hosts during registration.
The subscriptions that you add to an activation key must be available within the associated Content View.
```

> Subscription Manager attaches subscriptions differently depending on the following factors:

Reviewer: Subscription Manager does not attach subscriptions.
```suggestion
```

Subscription Manager attaches subscriptions differently depending on the following factors:

* Are there any subscriptions associated with the activation key?
* For {RHEL} 8 hosts: Is there system purpose set on the activation key?

Note that {Project} automatically attaches subscriptions only for the products installed on a host.
For subscriptions that do not list products installed on {RHEL} by default, such as the Extended Update Support (EUS) subscription, use an activation key specifying the required subscriptions.

> Based on the previous factors, there is one possible scenario for subscribing with activation keys:

Reviewer 2: ```suggestion
Based on the previous factors, there is a possible scenario for subscribing with activation keys:
```

> Based on the previous factors, there is one possible scenario for subscribing with activation keys:

Reviewer: All of this can be removed. Subscriptions are no longer attached to activation keys.

* Activation key that attaches subscriptions automatically.
+

> With no subscriptions specified, hosts using the activation key search for the best fitting subscription from the ones provided by the Content View associated with the activation key.

Reviewer: Remove this as well
```suggestion
```

> For {RHEL} 8 hosts, you can configure the activation key to set system purpose on hosts during registration to enhance the automatic subscriptions attachment.

Reviewer: You can still set system purpose values on an activation key, but the reason is different. Also it's not just for RHEL 8.
```suggestion
You can configure the activation key to set system purpose attributes on hosts during registration. This can help determine which repositories are available on the host, and also helps with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```

Note that {Project} automatically attaches subscriptions only for the products installed on a host.

> For subscriptions that do not list products installed on {RHEL} by default, such as the Extended Update Support (EUS) subscription, use an activation key specifying the required subscriptions and with the auto-attach disabled.

Reviewer: GitHub won't let me reference the lines above, but this entire section should be removed:

```
* Are there any subscriptions associated with the activation key?
* Is the auto-attach option enabled?
* For {RHEL} 8 hosts: Is there system purpose set on the activation key?
Note that {Project} automatically attaches subscriptions only for the products installed on a host.
For subscriptions that do not list products installed on {RHEL} by default, such as the Extended Update Support (EUS) subscription, use an activation key specifying the required subscriptions and with the auto-attach disabled.
```

> Activation key attaches subscriptions automatically.

Reviewer: Activation keys do not attach subscriptions.
```suggestion
```

> With no subscriptions specified and auto-attach enabled, hosts using the activation key search for the best fitting subscription from the ones provided by the Content View associated with the activation key.
> This is similar to entering the `subscription-manager --auto-attach` command.

Reviewer: Auto attach is not an option for activation keys now.
```suggestion
```

. From the *Environment* list, select the environment to use.
. From the *Content View* list, select a Content View to use.
ifndef::orcharhino[]
. In the *Repository Sets* tab, ensure only your named repository is enabled.
endif::[]
. Click *Save*.

> . Optional: For {RHEL} 8 hosts, in the *System Purpose* section, you can configure the activation key with system purpose to set on hosts during registration to enhance subscriptions auto attachment.

Reviewer: ```suggestion
. Optional: In the *System Purpose* section, you can configure the activation key with system purpose to set on hosts during registration.
```

> .. Click *Add Selected*.

Reviewer: This part needs removal:

```
. If SCA is not enabled:
.. Click the *Subscriptions* tab, then click the *Add* submenu.
.. Click the checkbox under the subscription you created before.
.. Click *Add Selected*.
```
